include/linux/regmap.h

```diff
@@ -835,6 +843,18 @@ bool regmap_ac97_default_volatile(struct device *dev, unsigned int reg);
 	__regmap_lockdep_wrapper(__regmap_init_sdw, #config,		\
 				sdw, config)
 
+/**
+ * regmap_init_sdw_mbq() - Initialise register map
```

Comment:
Initialize ?

Reply:
British spelling! 
I kept it since it's the same in the original regmap-sdw.

Reply:
thanks for education.  I always lost between British or American English